packages/webcomponents/src/components/payments-list/payments-list-core.tsx

```diff
@@ -71,9 +71,29 @@ export class PaymentsListCore {
     });
   };
 
+  handleDateChange = (name: string, value: string) => {
+    this.params = { ...this.params, [name]: value };
+  }
+
   render() {
     return (
       <Host>
+        <div class="row gy-3 mb-4">
+          <div class="col-2">
+            <form-control-date
+              name="created_after"
+              label="Date from:"
```

Comment:
I am not sure yet, but I think we should label them `Start Date` and `End Date`, with no `:`